terraform/modules/bap_env_gcp/variables.tf

```diff
@@ -21,6 +21,11 @@ variable "subnetwork" {
   default = "default"
 }
 
+variable "prefix" {
+  type        = string
+  description = "A prefix to add to the node name(s), e.g.: '<prefix>-<name>-x'"
```

Comment:
Shouldn't it refer to resource instead of node?

Reply:
I meant that it says `node` instead of `resource`, which I think it's more accurate.

Reply:
Upss